app/adapters/rating.js

```diff
@@ -0,0 +1,6 @@
+import DS from 'ember-data';
+
+export default DS.RESTAdapter.extend({
+  // host: 'http://skill-inventory.herokuapp.com',
```

Comment:
I took a look at this, too and couldn't think of anything different. Since it was already in the upstream master like this (commented out), I'm going to merge this commit in.